systemtest/src/main/java/io/strimzi/systemtest/utils/specific/MetricsUtils.java

```diff
@@ -130,7 +129,7 @@ public static void assertMetricResourcesLowerThanOrEqualTo(MetricsCollector coll
     public static void assertMetricResourcesIs(MetricsCollector collector, String kind, Predicate<Double> predicate, String message) {
         String metric = "strimzi_resources\\{kind=\"" + kind + "\",.*}";
         TestUtils.waitFor("metric " + metric + "is " + message, TestConstants.POLL_INTERVAL_FOR_RESOURCE_READINESS, TestConstants.GLOBAL_TIMEOUT_SHORT, () -> {
-            collector.collectMetricsFromPods();
+            collector.collectMetricsFromPods(TestConstants.GLOBAL_TIMEOUT);
```

Comment:
It is a little bit weird that collecting has timeout higher than dynamic wait. I would expect that collecting will try few times during the dynamic wait.

Reply:
updated